vassal-app/src/main/java/VASSAL/configure/BooleanConfigurer.java

```diff
@@ -35,6 +39,14 @@ public BooleanConfigurer(String key, String name) {
     this(key, name, Boolean.FALSE);
   }
 
+  public BooleanConfigurer (Boolean val) {
```

Comment:
I'm surprised checkstyle doesn't complain about the space between the name and the `(` here.